firmware/thruna/BMS/src/app/states/app_allStates.c

```diff
@@ -13,6 +13,16 @@
 // Num of cycles for voltage and cell temperature values to settle
 #define NUM_CYCLES_TO_SETTLE (30U)
 
+uint32_t owcCounter = 0;
+
+typedef enum
+{
+    RUN_CELL_MEASUREMENTS,
+    RUN_CELL_OWC
```

Comment:
To stay consistent with function name
```suggestion
    RUN_OPEN_WIRE_CHECK
```